resources/analysisTools/qcPipeline/bwaMemSortSlim.sh

```diff
@@ -273,4 +273,5 @@ mv ${FILENAME_QCJSON}.tmp ${FILENAME_QCJSON} || throw 27 "Could not move file"
 
 ${RSCRIPT_BINARY} ${TOOL_INSERT_SIZE_PLOT_SCRIPT} ${FILENAME_ISIZES_MATRIX} ${FILENAME_ISIZES_STATISTICS} ${FILENAME_ISIZES_PLOT}_temp "PE insertsize of ${bamname}" && mv ${FILENAME_ISIZES_PLOT}_temp ${FILENAME_ISIZES_PLOT} || ( echo "Error from insert sizes plotter" && exit 22)
 
-${RSCRIPT_BINARY} ${TOOL_PLOT_DIFFCHROM} -i "$FILENAME_DIFFCHROM_MATRIX" -s "$FILENAME_DIFFCHROM_STATISTICS" -o "${FILENAME_DIFFCHROM_PLOT}_temp" && mv  ${FILENAME_DIFFCHROM_PLOT}_temp ${FILENAME_DIFFCHROM_PLOT} || ( echo "Error from chrom_diff.r" && exit 23)
+FILENAME_DIFFCHROM_PLOT_TMP=$(getFilePathOnRoddyScratch "${FILENAME_DIFFCHROM_PLOT}")
+${RSCRIPT_BINARY} ${TOOL_PLOT_DIFFCHROM} -i "$FILENAME_DIFFCHROM_MATRIX" -s "$FILENAME_DIFFCHROM_STATISTICS" -o "${FILENAME_DIFFCHROM_PLOT_TMP}" && mv ${FILENAME_DIFFCHROM_PLOT_TMP} ${FILENAME_DIFFCHROM_PLOT} || ( echo "Error from chrom_diff.r" && exit 23)
```

Comment:
1. Quote subshell returns used as variable value
2. Split superlong line into multiline command.

```suggestion
FILENAME_DIFFCHROM_PLOT_TMP="$(getFilePathOnRoddyScratch "$FILENAME_DIFFCHROM_PLOT")"
$RSCRIPT_BINARY "$TOOL_PLOT_DIFFCHROM" \
  -i "$FILENAME_DIFFCHROM_MATRIX" \
  -s "$FILENAME_DIFFCHROM_STATISTICS" \
  -o "$FILENAME_DIFFCHROM_PLOT_TMP" \
  && mv "$FILENAME_DIFFCHROM_PLOT_TMP" "$FILENAME_DIFFCHROM_PLOT" \
  || ( echo "Error from chrom_diff.r" && exit 23)
```